Makefile

```diff
@@ -15,16 +15,19 @@ VARIANT ?= release
 # JerryScript options
 JERRY_BASE ?= $(ZJS_BASE)/deps/jerryscript
 EXT_JERRY_FLAGS ?= -DENABLE_ALL_IN_ONE=ON
+ifneq ($(DEV), ashell)
 ifeq ($(SNAPSHOT), on)
```

Comment:
This still won't work for 'make linux'.  I think this check / Flag set need to move to after ifeq ($(TARGET), linux).